src/IO/ADIOS/ADIOS2IOHandler.cpp

```diff
@@ -1107,7 +1107,9 @@ namespace detail
         adios2::Variable< T > var = m_impl->verifyDataset< T >(
             bp.param.offset, bp.param.extent, IO, bp.name );
 
-        engine.Put( var, ptr );
+        // default is Deferred, which is error prone (as of 2.6.0) and will 
```

Comment:
removing EOL whitespace:
```suggestion
        // default is Deferred, which is error prone (as of 2.6.0) and will
```